lib/scrabble_word.rb

```diff
@@ -0,0 +1,27 @@
+class ScrabbleWord
+
+  def initialize(word)
+    @word = word
+  end
+  def word
+    return @word
+  end
+  def score
+    letters = {
+      a: 1, b: 3, c: 3, d: 2, e: 1,
+      f: 4, g: 2, h: 4, i: 1, j: 8,
+      k: 5, l: 1, m: 3, n: 1, o: 1,
+      p: 3, q: 10, r: 1, s: 1, t: 1,
+      u: 1, v: 4, w: 4, x: 8, y: 4,
+      z: 10
+    }
+    total = 0
+    @word.downcase.split('').each do |letter|
+      total += letters[letter.to_sym]
+    end
+    return total
```

Comment:
I like how you are being explicit here with your returns, just know that the last line of each method is implicitly returned in Ruby